dialog.cpp

```diff
@@ -508,4 +508,13 @@ void Dialog::showConfigDialog()
     mConfigureDialog->exec();
 }
 
+
+/************************************************
+
+ ************************************************/
+void Dialog::reload()
+{
+    update(frameGeometry());
+}
+
```

Comment:
If I'm not missing something (I may, being busy with other codes right now), this can be changed to:

```c++
void Dialog::update()
{
    QWidget::update();
}
```
And then, it can be used above.